.travis.yml

```diff
@@ -44,6 +45,12 @@ jobs:
     script:
     - golint src/jetstream/...
     - ./deploy/ci/travis/update-go-report-card.sh
+  - stage: Build
```

Comment:
We could save ~4 minutes by combining this stage into the Lint one so they all run concurrently. OK to merge without this though